rmgpy/rmg/model.py

```diff
@@ -995,7 +1004,80 @@ def addSpeciesToEdge(self, spec):
         Add a species `spec` to the reaction model edge.
         """
         self.edge.species.append(spec)
-
+    
+    def setThermodynamicFilteringParameters(self,Tmax, toleranceThermoKeepSpeciesInEdge,minCoreSizeForPrune,maximumEdgeSpecies,reactionSystems):
+        """
+        sets parameters for thermodynamic filtering based on the current core
```

Comment:
You might want to describe what the inputs are (name, datatype, units, purpose) or something similar to other methods that fully describe the function. Maybe also mention that it returns nothing. It doesn't make much sense here, but may be more useful whenever the API gets updated.